cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -245,7 +243,8 @@ instance ElidingTracer (WithSeverity (ChainDB.TraceEvent blk)) where
       return (Just ev, count)
   conteliding _tverb _tr ev@(WithSeverity _ (ChainDB.TraceGCEvent _)) (_old, count) =
       return (Just ev, count)
-  conteliding _tverb tr ev@(WithSeverity _ (ChainDB.TraceLedgerReplayEvent (LedgerDB.ReplayedBlock pt [] replayTo))) (_old, count) = do
+                                                                                                                              -- TODO should this be used?
+  conteliding _tverb tr ev@(WithSeverity _ (ChainDB.TraceLedgerReplayEvent (LedgerDB.ReplayedBlock pt [] (LedgerDB.ReplayStart _replayFrom) (LedgerDB.ReplayGoal replayTo)))) (_old, count) = do
```

Comment:
There is an unused `_replayFrom` here.  Does it need to be used?

Reply:
@Jasagredo @EncodePanda